web-animations-1/Overview.bs

```diff
@@ -708,6 +701,22 @@ animation model to change, but these operations in themselves do not create
 a new [=animation frame=], rather they merely update the current <a>animation
 frame</a>.
 
+### Timeline Phase ### {#timeline-phases}
+
+A [=timeline=] may be in one of four possible 
+<dfn export lt="timeline phase">phases</dfn>:
+
+1.  <dfn export lt="timeline inactive phase">inactive</dfn>
+1.  <dfn export lt="timeline before phase">before</dfn>
+1.  <dfn export lt="timeline active phase">active</dfn>
+1.  <dfn export lt="timeline after phase">after</dfn>
+
+<div class="informative-bg"><em>This section is non-normative</em>
+
+[=timeline phase|Timeline phase=] is used to give additional context to 
+[=timeline current time=] when it would otherwise be ambiguous.
```

Comment:
This doesn't really explain anything. We should probably give an example.

Reply:
Removing this section until a later time when we can reference scroll timeline phases as an example.